docs/CDCBigTable-sparksink.md

```diff
@@ -1,16 +1,45 @@
-CDC Google Bigtable Sink
-==========
+# CDC Google Bigtable Sink
 
-CDAP Plugin for Change Data Capture (CDC) in Google Bigtable using the Spark Framework. Plugin can be configured only for real-time pipelines.
+Description
+-----------
+CDAP Realtime Plugin for Change Data Capture (CDC) in Google Bigtable. 
```

Comment:
This plugin takes input from a CDC source and writes the changes to Cloud Bigtable.

Reply:
fixed